.travis.yml

```diff
@@ -1,20 +1,33 @@
 dist: trusty
 language: go
-os:
-  - windows
-  - linux
-  - osx
-
-go:
-  - 1.12.x
-
 go_import_path: github.com/prometheus/tsdb
+matrix:
+  include:
+  - os: linux
+    dist: trusty
+    sudo: required
+    go: 1.12.x
+    install:
+      - make deps
+    script:
+      - sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install fuse
+      - sudo modprobe fuse
+      - sudo chmod 666 /dev/fuse
+      - sudo chown root:$USER /etc/fuse.conf
+      - sudo addgroup $USER fuse
+      - sudo chmod a+rw /etc/fuse.conf
+      - sudo echo user_allow_other >> /etc/fuse.conf
+      - sudo chmod +x /bin/fusermount
```

Comment:
Seems strange that this chmod really necessary

Reply:
Yeah, you are right, it is not necessary.
Fixed